src/data/equiv/mul_add.lean

```diff
@@ -475,6 +483,13 @@ lemma mul_left_symm_apply (a : G) : ((equiv.mul_left a).symm : G → G) = (*) a
 lemma mul_left_symm (a : G) : (equiv.mul_left a).symm = equiv.mul_left a⁻¹ :=
 ext $ λ x, rfl
 
+@[to_additive]
+lemma mul_left_bijective (a : G) : function.bijective ((*) a) :=
+begin
+  rw ← coe_mul_left,
```

Comment:
Are you sure you need the rewrite?

Reply:
Its not necessary, just seemed nicer to use the lemma rather then rely on defeq? I've removed it now